deno.jsonc

```diff
@@ -29,14 +29,14 @@
   },
   "tasks": {
     "run": "deno run -A",
-    "run:browser": "deno task run test_util/ctx.ts -- deno task run _tasks/run_browser.ts",
     "debug": "deno task run --inspect-brk",
     "udd": "deno task star && deno task run https://deno.land/x/udd@0.5.0/main.ts target/star.ts",
     "dnt": "deno task run _tasks/dnt.ts",
     "star": "deno task run _tasks/star.ts && deno cache --check target/star.ts",
     "codegen": "deno task run cache.ts examples/mod.ts",
     "test": "deno task run test_util/ctx.ts -- deno test -A -L=info --ignore=target --parallel",
     "test:update": "deno task test -- -- --update",
+    "test:browser": "deno task run --unstable _tasks/test_browser/main.ts",
```

Comment:
Why is the `--unstable` flag necessary?

Reply:
Seems like it works without an `--unstable` . Removed.
- https://github.com/lucacasonato/deno-puppeteer#usage

- https://alexandrempsantos.com/deno/puppeteer-with-deno/
> Due to deno-puppeteer using a few Deno APIs that are still under active development (and subject to changes), we will need to run our program with the --unstable flag.